ova/Vagrantfile

```diff
@@ -7,8 +7,8 @@ Vagrant.configure("2") do |config|
 
   config.vm.provider "virtualbox" do |vb|
     vb.name = "vm_wazuh"
-    vb.memory = "4096"
+    vb.memory = "5120"
```

Comment:
Revert this change the OVA should have 4GB of RAM.

Reply:
Done in 71fd88cf61ad859245d50e315178f0058cb61754